paper-server/patches/sources/net/minecraft/server/MinecraftServer.java.patch

```diff
@@ -1027,8 +1027,9 @@
 +                }
 +                ServerPlayer entityplayer = (ServerPlayer) entityhuman;
 +                long playerTime = entityplayer.getPlayerTime();
-+                ClientboundSetTimePacket packet = (playerTime == dayTime) ? worldPacket :
-+                    new ClientboundSetTimePacket(worldTime, playerTime, doDaylight);
++                boolean relativeTime = entityplayer.relativeTime;
++                ClientboundSetTimePacket packet = ((relativeTime || !doDaylight) && playerTime == dayTime) ? worldPacket :
```

Comment:
What is the point in also checking the negated doDaylight here?

Just a performance improvement to potentially reuse the world packet?